server/conf/env-var-docs.json

```diff
@@ -703,6 +703,24 @@
     "description": "The amount of time, in minutes, that a session lasts. The default is 600 minutes, or 10 hours. Note that there isn't yet messaging on the frontend to notify a user when their session is expired.",
     "type": "int"
   },
+  "SESSION_INACTIVITY_WARNING_MINUTES": {
+    "mode": "ADMIN_READABLE",
+    "description": "The number of minutes of inactivity before we warn the user that their session will expire. Default is 5.",
```

Comment:
question: Does this mean we will warn the user after 5 minutes of inactivity or 5 minutes before their session expires due to inactivity? The language is a little unclear.

Reply:
How about this:
"How long before the inactivity timeout, in minutes, a user receives a warning that their session is about to expire."

Reply:
updated the wording.

Reply:
This is much clearer. Thank you!